scripts/setfit/run_fewshot.py

```diff
@@ -99,6 +101,19 @@ def main():
     for dataset, metric in dataset_to_metric.items():
         few_shot_train_splits, test_data = load_data_splits(dataset, args.sample_sizes, args.add_data_augmentation)
 
+        # Potentially report on naive use of accuracy with an imbalanced test set
+        if metric == "accuracy":
```

Comment:
Do you think we throw a warning whenever there is an unbalanced dataset is a good idea?

For me I would like to know any unbalanced dataset and the corresponding metric. Specifically, maybe we can take off this if statement to allow throwing warnings on other datasets that are unbalanced but with different metrics than "accuracy"?